src/pages/Map.js

```diff
@@ -56,6 +76,7 @@ export default function Map() {
                 locationData={locationData}
                 showWindow={showWindow}
                 closeWindow={setShowWindow}
+                zIndex={2}
```

Comment:
I'm curious why this was necessary? What behavior were you seeing without the `zIndex`?

Reply:
That was my suggestion! I hardcoded for the center of my town & couldn't find the "you are here" icon because it was underneath the pins. I suggested we make it z-index 1 and bigger. However, I'm open to other ways.

Reply:
Ah, got it. Makes sense!